raiden/raiden_service.py

```diff
@@ -561,6 +630,8 @@ def close(self, token_address, partner_address):
             second_transfer,
         )
 
+        return netting_channel
```

Comment:
We should rename the RaidenAPI-class, since this is very confusing terminology once all the other API-classes are introduced. 
From my understanding RaidenAPI is more of a Command-Line-Tool, that could also go into the RaidenService, but was abstracted to another ("circular-dependency-") class.
So I would still consider it "internal".
The actual API-classes wrap around RaidenAPI and handle encoding of objects.
I had the same feeling, that the interface for `REST` users and `library` users should be the same, and both should provide the same amount of information.
Once we want to provide the library, we could introduce an additional replacement-class for the [RestAPI](https://github.com/LefterisJP/raiden/blob/b8262bb56652b8947e6cc405e09550b2240c6bfe/raiden/api/rest.py#L82), that doesn't encode to JSON, but rather converts to [data-container-objects](https://github.com/LefterisJP/raiden/blob/b8262bb56652b8947e6cc405e09550b2240c6bfe/raiden/api/objects.py#L25), with the same interface and amount of information as the JSON-Objects.